docker/build-aws.sh

```diff
@@ -75,8 +75,14 @@ submit_build() {
 }
 
 get_build_status() {
+    BUILD_STATUS=""
+    CURRENT_PHASE=""
     local BUILD_ID="${1}"
     local BUILD_JSON=$(aws codebuild batch-get-builds --ids ${BUILD_ID})
+    if [[ $? -gt 0 ]]; then
```

Comment:
Did you check this actually works? At least in command line exit status here is `1`, because it's exit status of assignment statement (X=... always succeeds)
```
achursin-mbp:libra achursin$ X=$(touch /1); echo $?
touch: /1: Read-only file system
1
```